problems/concat/solution.js

```diff
@@ -4,5 +4,5 @@
   
   process.stdin.pipe(concat(function (src) {
       var s = src.toString().split('').reverse().join('');
-      console.log(s);
+      process.stdout.write(s);
```

Comment:
I do the other changes. But with this I doubt:
 * in the problem.txt don't mention to add a line break to the reversed stdin 
 * the tests pass
 * in my opinion is less elegant (to add a "+'\n' " or something to add a character to the stdout. Think that the output has 1 char more than the input).